docs/getting-started/integrations-cdn/vuejs-v3.md

```diff
@@ -8,150 +8,114 @@ order: 50
 
 ```html
 <template>
-	<div id="app">
-		<ckeditor
-			v-model="editorData"
-			:editor="editor"
-			:config="editorConfig"
-		/>
-	</div>
+	<ckeditor
+		v-if="editor"
+		v-model="data"
+		:editor="editor"
+		:config="config"
+	/>
 </template>
 
 <script>
-import { ClassicEditor, Bold, Essentials, Italic, Mention, Paragraph, Undo } from 'ckeditor5';
-import { SlashCommand } from 'ckeditor5-premium-features';
-
-import 'ckeditor5/ckeditor5.css';
-import 'ckeditor5-premium-features/ckeditor5-premium-features.css';
+import { Ckeditor, useCKEditorCloud } from '@ckeditor/ckeditor5-vue';
 
 export default {
-	name: 'app',
+	name: 'Editor',
+	components: {
+		Ckeditor
+	},
 	data() {
 		return {
-			editor: ClassicEditor,
-			editorData: '<p>Hello from CKEditor 5 in Vue!</p>',
-			editorConfig: {
-				licenseKey: '<YOUR_LICENSE_KEY>', // Or 'GPL'.
-				plugins: [ Bold, Essentials, Italic, Mention, Paragraph, SlashCommand, Undo ],
-				toolbar: [ 'undo', 'redo', '|', 'bold', 'italic' ],
-				// Other configuration options...
+			cloud: useCKEditorCloud( {
+				version: '43.0.0',
+				premium: true
+			} ),
+			data: '<p>Hello world!</p>',
+			config: {
+				licenseKey: '<YOUR_LICENSE_KEY>', // Or "GPL"
+				toolbar: [ 'heading', '|', 'bold', 'italic' ]
 			}
-		};
+		}
+	},
+	computed: {
+		editor() {
+			if ( !this.cloud.data ) {
+				return null;
+			}
+
+			const {
+				ClassicEditor,
+				Paragraph,
+				Essentials,
+				Heading,
+				Bold,
+				Italic,
+				Mention
+			} = this.cloud.data.CKEditor;
+
+			const { SlashCommand } = this.cloud.data.CKEditorPremiumFeatures;
+
+			return class Editor extends ClassicEditor {
+				static builtinPlugins = [
+					Essentials,
+					Paragraph,
+					Heading,
+					Bold,
+					Italic,
+					Mention,
+					SlashCommand
+				];
+			};
+		}
 	}
-};
+}
 </script>
 ```
 
-#### Using the `<ckeditor>` component locally
-
-If you do not want to enable the CKEditor&nbsp;5 component globally, you can import the `Ckeditor` component from the `@ckeditor/ckeditor5-vue` package directly into the Vue component where you want to use it, and add it to the `components` object.
+Now you can import and use the `Editor.vue` component anywhere in your application.
 
 ```html
 <template>
-	<div id="app">
-		<ckeditor
-			v-model="editorData"
-			:editor="editor"
-			:config="editorConfig"
-		/>
-	</div>
+	<Editor />
 </template>
+```
 
-<script>
-import { ClassicEditor, Bold, Essentials, Italic, Mention, Paragraph, Undo } from 'ckeditor5';
-import { SlashCommand } from 'ckeditor5-premium-features';
-import { Ckeditor } from '@ckeditor/ckeditor5-vue';
-
-import 'ckeditor5/ckeditor5.css';
-import 'ckeditor5-premium-features/ckeditor5-premium-features.css';
+If you use Nuxt.js with server-side rendering enabled, remember to wrap the `<Editor>` component in the `<ClientOnly>` component to avoid issues with the editor calling browser-specific APIs on the server.
```

Comment:
I guess the extra section is now enough for Nuxt. But, in the future, we should probably add a full-fledged guide.